.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,49 @@
-- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
+<!--
+Thank you for sending the PR! 
+Please fill the applicable details below
+Happy contributing!
+-->
 
-- **Why was this change needed?** (You can also link to an open issue here)
+### What kind of change does this PR introduce? <!-- Mark The Boxes -->
 
-- **Other information**:
+- [ ] Bug
+- [ ] Feature
+- [ ] Docs
+- [ ] Other(s)
```

Comment:
I like this addition! It will help make it clearer for people who open a pr that they need to mention it and for us to immediately see as it will be in the same format all the time

Reply:
Yeah, I guess this could make this question easier to answer "What kind of change does this PR introduce?" 😄